public/app/plugins/datasource/loki/types.ts

```diff
@@ -33,6 +34,8 @@ export interface LokiQuery extends DataQuery {
   resolution?: number;
   range?: boolean;
   instant?: boolean;
+  // extra info passed to Loki API when log volume query is run
```

Comment:
I think this comment should be next to `hint` param in the `LokiRangeQueryRequest`

Reply:
Updated in eb618d74b4d6caf72e789f02097d42fd00098520